docs/ref/settings.txt

```diff
@@ -2258,13 +2258,46 @@ The secret key is used for:
 
 If you rotate your secret key, all of the above will be invalidated.
 Secret keys are not used for passwords of users and key rotation will not
-affect them.
+affect them. Use :setting:`SECRET_KEYS` to allow secret key rotation.
```

Comment:
I get that we don't want to deprecate `SECRET_KEY` at this stage, but do we want to add an admonition that encourages use of `SECRET_KEYS` more strongly now? This would, hopefully, make it easier to do a more strict deprecation later, if that decision is made, as we've highlighted the preferred setting as early as possible to encourage wider use.